kafka/coordinator/consumer.py

```diff
@@ -31,7 +32,7 @@ class ConsumerCoordinator(BaseCoordinator):
         'enable_auto_commit': True,
         'auto_commit_interval_ms': 5000,
         'default_offset_commit_callback': None,
-        'assignors': (RangePartitionAssignor, RoundRobinPartitionAssignor),
+        'assignors': (RangePartitionAssignor, RoundRobinPartitionAssignor, StickyPartitionAssignor),
```

Comment:
would rather not add this to the default list, especially since it requires an external dependency

Reply:
This assignment algorithm isn't just sticky, it's also better balanced than either of the other two... and it favors better balancing over stickiness... at least that's what I recall from when I helped drive that KIP forward.

So I think in that case, it'd be nice to add to the default list, and I'd even support moving it up the list to the primary default so that users get the most optimized balancing algorithm.

Be good to know what the Java client does here, I'd check but my wife just said dinner is ready so I need to roll...

Reply:
I looked into this a little more, and based on the work in [KIP-429](https://cwiki.apache.org/confluence/display/KAFKA/KIP-429%3A+Kafka+Consumer+Incremental+Rebalance+Protocol), it's probably not a great idea to set this as the preferred option... 

But I do still support including it in the list of default options. Additionally, if this is joining a mixed-language consumer group, it's more likely that it'd be cross-compatible with the Java clients etc if we include this in the default list.